tests/queries/models.py

```diff
@@ -737,3 +737,57 @@ class RelatedIndividual(models.Model):
 
     class Meta:
         db_table = 'RelatedIndividual'
+
+
+class SchemaQualified(models.ModelTable):
+    def __init__(self, schema, table):
+        self.schema = schema
+        self.table = table
+
+    def as_sql(self, compiler, connection):
+        # Note that it would be easy to change the schema using thread locals
+        # if wanted.
```

Comment:
So the intent here is that we could have dynamic schema selection, based on some external factor? For instance, django-boardinghouse (my project) allows setting the schema as part of the session. Shared models/tables exist only in the public/main schema, and all others are in all client schemata.